xonsh/readline_shell.py

```diff
@@ -43,6 +43,7 @@
 RL_VARIABLE_VALUE = None
 _RL_STATE_DONE = 0x1000000
 _RL_STATE_ISEARCH = 0x0000080
+_RL_PREV_CASE_SENSITIVE_COMPLETIONS = None
```

Comment:
hmm, maybe I better to set this guy to something other than `None`, like a str `'to-be-set'`. Because `env.get()` could get `None` and `readline.parse_and_bind` would not be called (it should better to be called at least once). @scopatz thoughts here?